CRBM-Viz/src/environments/environment.ts

```diff
@@ -2,9 +2,9 @@ export const environment = {
   production: false,
   baseUrl: '/',
   auth0: {
-    domain: 'crbm.auth0.com',
-    clientId: '0NKMjbZuexkCgfWY3BG9C3808YsdLUrb',
-  },
```

Comment:
Please dont edit the default the environment! Can you use dev for this, or is there a particular reason you need it to be the default?